.azure-pipelines/azure-pipelines.yml

```
@@ -134,6 +134,8 @@ stages:
               test: fedora37
             - name: Ubuntu 20.04
               test: ubuntu2004
+            - name: Ubuntu 22.04
```

```suggestion
            - name: Ubuntu 22.04
```
Should we also add this to test against `- stage: Docker_devel` ?